src/M6Web/Bundle/WSClientBundle/Adapter/Response/GuzzleResponseAdapter.php

```diff
@@ -95,4 +95,22 @@ public function getHeaderValue($header, $glue = '')
         return $header;
     }
 
+    /**
+     *  Magic method to the Response adapter
+     *
+     * @param string $name      method name
+     * @param array  $arguments method arguments
+     *
+     * @throws Exception
+     * @return mixed
+     */
+    public function __call($name, $arguments)
+    {
+        if (!method_exists($this->response, $name)) {
+            throw new BadMethodCallException("Method ".$name." doesn't exist in Guzzle Response");
```

Comment:
Not only `GuzzleResponse` ! All class who implements `GuzzleHttp\Message\ResponseInterface`